public/app/features/admin/ldap/LdapSyncInfo.tsx

```diff
@@ -28,36 +28,33 @@ export class LdapSyncInfo extends PureComponent<Props, State> {
     const { isSyncing } = this.state;
```

Comment:
This syncing doesn't seem to do anything except altering UI state. Can it be removed? Should I also refactor this into a function component?